config/puma.rb

```diff
@@ -7,9 +7,8 @@
 threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
 threads threads_count, threads_count
 
-# Specifies the `port` that Puma will listen on to receive requests; default is 3000.
-#
-port        ENV.fetch("PORT") { 3000 }
+# Specifies the socket Puma will listen on, can be tcp:// or unix://, defaults to localhost:3000
+bind ENV.fetch("BIND") { "tcp://[::1]:3000" }
```

Comment:
You could probably still respect the PORT env var here, untested code, so worth checking, but maybe something like this
```suggestion
bind ENV.fetch("BIND") { "tcp://[::1]:#{ ENV.fetch('PORT') || '3000' }" }
```

Reply:
Hi, thanks for the idea, I guess it would make sense for backwards compatibility, but it does not work in my testing, at least not without having `PORT` in the environment:

```
puma.rb:17:in `fetch': key not found: "PORT" (KeyError)
```

Reply:
This seems to load:

```
bind ENV.fetch("BIND") { "tcp://[::1]:#{ ENV.fetch('PORT') { '3000' } }" }
```

Reply:
I change the way it fetches default, that may work a bit better

Reply:
Indeed, that seems to work similar to my variant, but a bit cleaner.